projects/plugins/jetpack/tests/e2e/specs/waf/blocking.test.js

```diff
@@ -0,0 +1,31 @@
+import { test, expect } from 'jetpack-e2e-commons/fixtures/base-test.js';
+import { WpPage } from 'jetpack-e2e-commons/pages/index.js';
+import playwrightConfig from '../../playwright.config.cjs';
+import { Plans, prerequisitesBuilder } from 'jetpack-e2e-commons/env/index.js';
+import { resolveSiteUrl } from 'jetpack-e2e-commons/helpers/utils-helper.cjs';
+
+test.describe.parallel( 'WAF Blocking', () => {
```

Comment:
Is there a particular reason to use `parallel` here?

Reply:
No, we probably oriented that too exactly on other examples in the repo :)

Reply:
Just to be sure before I change it: Is it problematic that we call this parallel? I'm thinking that someone adding tests to this later on may forget to do so when it would indeed be a good idea to save execution time?

Reply:
> Is it problematic that we call this parallel?

Not at all. At least at this point. When running tests in parallel, the same site state (connection, plan, browser session etc) will be reused. If one of the tests will modify that (i.e. you'll add a WAF test for not logged in user), it will affect other tests in the same spec file.